torch/autograd/profiler.py

```diff
@@ -101,55 +101,49 @@ def export_chrome_trace(self, path):
         Arguments:
             path (str): Path where the trace will be written.
         """
-        import json
+        import os
         with open(path, 'w') as f:
             chrome_events = []
             next_id = 0
+            f.write("[")
```

Comment:
Someone is definitely going to ask you why you're building json with string concatenation, there should definitely be a comment saying why :) :) :)

Reply:
Alright.